doc/manual.rst

```diff
@@ -4166,6 +4166,112 @@ For the above `openArray` and `var` rules, whether the expressions refers to
 a location derived from a parameter or global must be visible directly from the
 argument expression, no indirection through locals is allowed.
 
+Delimited Continuations
+-----------------------
+
+For reifying a continuation, that is, turning a conceptual continuation into a
+concrete first-class value (a *continuation instance*), the `suspend`:idx: form
+is used.
+
+A continuation instance is the reified continuation (a procedure) plus the
+saved state needed to run it.
+
+Every available call syntax may be used for `suspend`, including the command
+and dot-call syntax. `suspend` is not a real procedure and does not participate
+in overload resolution, but the name in the callee position may be qualified
+as `system.suspend`.
```

Comment:
This will have to be clarified in terms of overloading resolution.

What would "not participate in overload resolution" mean? Does that mean this is basically a new keyword? Or is it more that "when `system.suspend` is in scope, it always win"? What happens if `import system except suspend` is used?

Reply:
It works the same as `addr`, `sizeof`, `quote`, and other pseudo-keywords do.